### DISCUSSION TITLE
Roadmap?

### DISCUSSION THREAD

**Discussion by fedotovartuom76** [jarvis394/geekr#169] [General]
I would like to see the roadmap of the project to understand when to expect the implementation of any new features.

**Comment by jarvis394** [comment]
Here you can see some of the points I still haven't done: https://habr.com/ru/post/575210#dalneyshee-razvitie

Will probably upload full list of points for myself in some near future (still have lots of work at uni 😥)

**Reply by fedotovartuom76** [reply]
Thank you, I saw this post, but unfortunately it does not contain even approximate implementation dates. I understand that you have a lot of work to do and I'm willing to wait. But as a user, I would like to know at least an approximate implementation date for these features.

**Comment by jarvis394** [comment]
@fedotovartuom76 Roadmap can now be accessed here: [Notion page](https://tarnatovski.notion.site/581dce00a7af40cfa0b978a77aa5e299?v=afaccee230ce440498326acea7e4e936)

![image](https://user-images.githubusercontent.com/37776763/156647638-dad8b469-0ff1-48b9-b9ff-ec20b4e23cb9.png)

Obviously, this is not complete list for now but at least it has been started.

**Reply by fedotovartuom76** [reply]
Thank you, this is already a start. I hope you will continue to develop the project in your spare time.
